lib/src/migrators/module.dart

```diff
@@ -1,41 +1,30 @@
-// Copyright 2018 Google LLC
+// Copyright 2019 Google LLC
 //
 // Use of this source code is governed by an MIT-style
 // license that can be found in the LICENSE file or at
 // https://opensource.org/licenses/MIT.
 
-import 'dart:io';
-
 // The sass package's API is not necessarily stable. It is being imported with
 // the Sass team's explicit knowledge and approval. See
 // https://github.com/sass/dart-sass/issues/236.
 import 'package:sass/src/ast/sass.dart';
+import 'package:sass/src/syntax.dart';
 
-import 'package:args/args.dart';
 import 'package:path/path.dart' as p;
 
-import 'package:sass_migrator/src/migrator_base.dart';
+import 'package:sass_migrator/src/migration_visitor.dart';
+import 'package:sass_migrator/src/migrator.dart';
 import 'package:sass_migrator/src/patch.dart';
 import 'package:sass_migrator/src/utils.dart';
 
 import 'module/built_in_functions.dart';
 import 'module/local_scope.dart';
-import 'module/stylesheet_migration.dart';
 
-class ModuleMigrator extends MigratorBase {
+/// Migrates stylesheets to the new module system.
+class ModuleMigrator extends Migrator {
   final name = "module";
+  final aliases = ["modules", "module-system"];
```

Comment:
I'm personally not a fan of adding command aliases unless you have to for backwards compatibility (or puns, like `pub lish` :stuck_out_tongue_winking_eye:), but it's up to you.